src/runtime/lpython_builtin.py

```diff
@@ -486,3 +486,21 @@ def _mod(a: i64, b: i64) -> i64:
 @overload
 def _mod(a: f64, b: f64) -> f64:
     return a - _lpython_floordiv(a, b)*b
+
+@overload
+def max(a:i32 , b:i32) ->i32:
+    if a > b :
+        return a
+    else :
+        return b
+@overload
+def max(a:i32 , b:i32 , c:i32)->i32:
```

Comment:
I think `max` can accept any number of arguments and not necessarily 2 and 3. Overloading for any new number of arguments is fine for now but in distant future we should support variadic arguments.

Reply:
yes that was my thinking first ,Ondrej told me to just make it with three arguments as a maximum for now.

Reply:
i want to make arguments containing (char , int , float ) but this will make alot of overloaded functions

Reply:
Yes, that's totally fine to make `max` accept at most 3 arguments. I would suggest to overload `max` for `char`, `int`, `float` types with two arguments. See below. Then we will overload further as needed in future.

```python
@overload
def max(a: char, b: char):
    ...

@overload
def max(a: float, b: float):
    ...
```

Reply:
yes , i making it like that now, can i use list for arguments insted of three or two variables ?

Reply:
> can i use list for arguments

Is it supported in LPython as of now?

Reply:
i don't know ,but i saw lists in `lower_bound()` in builtin files, but it doesn't has implementation.